src/net/sourceforge/kolmafia/session/DaylightShavingsHelmetManager.java

```diff
@@ -0,0 +1,30 @@
+package net.sourceforge.kolmafia.session;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import net.sourceforge.kolmafia.preferences.Preferences;
+
+public class DaylightShavingsHelmetManager {
+  public static final Pattern[] DAYLIGHT_SHAVINGS_PATTERNS = {
+    Pattern.compile(
+        "Your helmet shoots some lasers at your face\\.  You smell burning hair\\.<center><table><tr><td><img class=hand src=\"\\/images\\/itemimages\\/dsstache\\d+\\.gif\" onClick='eff\\(\"[^\"]+\"\\);' width=30 height=30 alt=\"([^\"]+)\""),
```

Comment:
With `\\.  ` the common way to handle it in mafia is `\\. +` to cover single and double space weirdness